### ISSUE TITLE
Motospeed  ck-61

### ISSUE THREAD

**Issue by ppattern** [ah-/anne-key#71]
Motospeed ck-61 missing tilde and grave.
Can I use anne-key with it.

**Comment by DonaldTsang** [comment]
If I am allowed to make a list, here are all the keyboards that are 61 key and is "cheap"
- [ ] Royal Kludge RK61
- [ ] Motospeed CK61 and CK62
- [ ] Geek GK61
- [ ] Ajazz i610T
- [ ] Maibenben DMK01

For those of similar price
- [ ] Geek Customized SK61
- [ ] FEKER
- [ ] IQUNIX F60

**Comment by MycroftHolmes1989** [comment]
I don't think this project can be extended to cover those boards. This particular firmware is aimed for Anne Pro. The difference in MCU prevent it being extended to even Anne Pro 2.
